src/module/api/index.ts

```diff
@@ -0,0 +1,5 @@
+import * as update from "./update";
+
+export const API = {
+  Update: update.Update,
+};
```

Comment:
The api module is the entrypoint for the webclient to talk to the remote api. The api module defines services that wrap functionality around resources. Here a resource is e.g. an Update or a Timeline. Here only the Update resource is introduced.